### DISCUSSION TITLE
Link to the game itself from readme

### DISCUSSION THREAD

**Discussion by hovi** [MaxRobinsonTheGreat/EvolutionSimulatorV2#143] [Ideas and Suggestions]
Small tip - link from the readme to the game itself, ideally somewhere at start. I even cloned it first to play it cause I thought it's not deployed anywhere, then I found in discussions, that it's actually at https://thelifeengine.net/
Make it more obvious, sell this cool project better is what I say! :)

**Comment by aaron-jencks** [comment]
You could do this yourself with a PR
